Sources/SwifterSwift/SwiftStdlib/KeyedDecodingContainer.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  KeyedDecodingContainer.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+#if canImport(Foundation)
+import Foundation
+#endif
```

Comment:
`KeyedDecodingContainer` is defined in `Foundation`, please wrap the entire file with `#if canImport(Foundation)`

> If you do so, no need to wrap each function separately 

```swift
#if canImport(Foundation)
import Foundation

public extension KeyedDecodingContainer {
   ...
}
#endif
```

Reply:
`KeyedDecodingContainer` is part of the Swift stdlib. The reason that `Foundation` is being used in these new functions is in order to use `NSString` and `NSInteger`'s `.boolValue` properties, which look for one of several predefined values meaning `true`.

Reply:
Ohh, sorry my bad 😅